infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -470,15 +469,122 @@ https://github.com/organizations/{YOUR ORG}/settings/installations/{INSTALLATION
      - `PSOXY_GITHUB_PRIVATE_KEY` with content of the `gh_pk_pkcs8.pem` from previous step. You could open the certificate with VS Code or any other editor and copy all the content *as-is* into this variable.
   7. Once the certificate has been uploaded, please remove {YOUR DOWNLOADED CERTIFICATE FILE} and `gh_pk_pkcs8.pem` from your computer or store it in a safe place.
 
+EOT
+    }
+    github-enterprise-server = {
+      source_kind : "github",
+      worklytics_connector_id : "github-enterprise-server-psoxy"
+      display_name : "Github Enterprise Server"
+      identifier_scope_id : "github"
+      worklytics_connector_name : "Github Enterprise Server via Psoxy"
+      target_host : var.github_enterprise_server_host
+      source_auth_strategy : "oauth2_refresh_token"
+      secured_variables : [
+        {
+          name : "ACCESS_TOKEN"
+          writable : true
+          sensitive : true
+          value_managed_by_tf : false
+        },
+        {
+          name : "PRIVATE_KEY"
+          writable : false
+          sensitive : true
+          value_managed_by_tf : false
+        },
+        {
+          name : "CLIENT_ID"
+          writable : false
+          sensitive : true
+          value_managed_by_tf : false
+        },
+        {
+          name : "OAUTH_REFRESH_TOKEN"
+          writable : true
+          lockable : true
+          sensitive : true
+          value_managed_by_tf : false
+        }
+      ],
+      environment_variables : {
+        GRANT_TYPE : "certificate_credentials"
+        TOKEN_RESPONSE_TYPE : "GITHUB_ACCESS_TOKEN"
+        REFRESH_ENDPOINT : "https://${var.github_enterprise_server_host}/app/installations/${local.github_installation_id}/access_tokens"
+        USE_SHARED_TOKEN : "TRUE"
+      }
+      settings_to_provide = {
+        "GitHub Organization" = local.github_organization
```

Comment:
@eschultink do we need to support multiple orgs per connector? Make this a csv / array?

Reply:
in theory, yeah ... but they could just put is a CSV, right? should work. Still a string.